.github/workflows/deployDocumentation.yml

```diff
@@ -0,0 +1,53 @@
+name: gh-documentation
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [documentation]
```

Comment:
It probably needs to be changed here as well, since we decided to have de documentation in `main`, right?

Reply:
@didiladi  I am leaving that on purpose because in the future if I push to the documentation branch, I want to see what the gh page looks like and make changes before making pull request to the main branch

Reply:
Ah, I see. How is this then working? Does a push to the `documentation` branch temporarily override the one coming from `main`, or is it published on some temporary location?

Reply:
yes it overrides the one coming from main temporarily